manage.py

```diff
@@ -6,7 +6,7 @@
 
 if __name__ == "__main__":
     dotenv.read_dotenv()
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "waterquality.settings")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings.dev")
```

Comment:
not sure about this but it wouldn't run when the settings file didn't exist

Reply:
error was ```  File "/Users/david/.virtualenvs/venv/lib/python2.7/site-packages/django/conf/__init__.py", line 110, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named settings```

Reply:
That makes sense. You inserted the correct settings path